### PR TITLE
Add lastRefreshTime to UserMetadata toJSON method.

### PR DESCRIPTION
**Pull request by rsgowman**
See #887

### CODE REVIEW

**Comment by rsgowman** [comment]
Note the diffbase. (Will retarget to master after that other one lands)

**Comment by rsgowman** [comment]
> This is confusing. Why not put all the fixes in one PR. They are closely related.

go/small-cls? But more relevantly, my fix for the importing one isn't working and will take a bit longer than these two more trivial ones.

File: test/unit/auth/user-record.spec.ts [hunk 1]
```diff
@@ -80,6 +80,7 @@ function getValidUserResponse(tenantId?: string): GetAccountInfoUserResponse {
     validSince: '1476136676',
     lastLoginAt: '1476235905000',
     createdAt: '1476136676000',
+    lastRefreshAt: '2016-10-12T01:31:45.000Z',
```
[the remark this answers is not in the document]
Comment (bojeil-google) [reply test/unit/auth/user-record.spec.ts:83]: Yeah they are all sent as strings. Here is a snippet:
```
      "lastLoginAt": "1590622314664",
      "createdAt": "1590545248026",
      "lastRefreshAt": "2020-05-28T01:44:39.262Z"
```

**Comment by cainenielsen** [comment]
@rsgowman do you know the status of this fix? I am using the RTDB Admin SDK and am still not receiving the lastRefreshTime.

**Comment by rsgowman** [comment]
@cainenielsen sorry, looks like this stalled out. Let me see if I can update it and find the right person to approve it.

**Comment by rsgowman** [comment]
@lahirumaramba, could you review this old PR (or redirect appropriately)? Thanks!

**Review by lahirumaramba — APPROVED** [review]

**Comment by lahirumaramba** [comment]
@rsgowman Thank you for surfacing this. LGTM. I will merge this.

**Pull request merged**